app/models/pick.rb

```diff
@@ -0,0 +1,8 @@
+class Pick < ActiveRecord::Base
+  validates :user_id, presence: true
```

Comment:
I just tested this on my computer and you can remove the `_id` suffix here and in the test and all tests run fine. Slightly cleaner IMO too.